src/tools/rbd/action/Kernel.cc

```diff
@@ -188,6 +221,10 @@ static int do_kernel_map(const char *poolname, const char *imgname,
 
   free(devnode);
 out:
+  if (r < 0) {
+    print_error_description(poolname, imgname, snapname, -r);
```

Comment:
It won't (or very unlikely to).  The only call we want to try to interpret errors and/or direct users to syslog for is the map call - everything else is going to be different.